blueocean-core-js/src/js/index.js

```diff
@@ -90,17 +100,15 @@ sseService.registerHandler(defaultSSEhandler.handleEvents);
 
 // Export some debugging stuff client code may need
 
-import { enableMocksForI18n, disableMocksForI18n } from './i18n/i18n';
-
 export const DEBUG = {
     enableMocksForI18n,
     disableMocksForI18n,
 };
 
+export { BunkerService } from './services/BunkerService';
```

Comment:
@cliffmeyers I did need to export this type so I can use `BunkerService` itself in `blueocean-dashboard`